rule_packages/c/IO4.json

```diff
@@ -0,0 +1,71 @@
+{
+  "CERT-C": {
+    "FIO45-C": {
+      "properties": {
+        "obligation": "rule"
+      },
+      "queries": [
+        {
+          "description": "TOCTOU race conditions when accessing files can lead to vulnerability.",
+          "kind": "problem",
+          "name": "Avoid TOCTOU race conditions while accessing files",
+          "precision": "high",
+          "severity": "error",
+          "short_name": "ToctouRaceConditionsWhileAccessingFiles",
+          "tags": [
+            "correctness",
+            "security"
+          ],
+          "implementation_scope": {
+            "description": "The query is limited to the specific class of TOCTOU race conditions."
```

Comment:
one more little nitpick, in this description do we want to explicitly say which file operations the "specific class" we handle covers?
its just that the wording in the rule description page for this is so broad: "performs two or more file operations "